packages/apollo/src/components/OrdererDetails/ChannelParticipationDetails.js

```diff
@@ -185,6 +636,29 @@ class ChannelParticipationDetails extends Component {
 						widerTiles
 					/>)
 				}
+
+				{!this.props.loading && this.props.showEditChannelModal && (
```

Comment:
when the pencil button is clicked, there is no obvious loading animation and the `ChannelModal` is still hidden, it will eventually appear and all is well, but this is confusing. change this to show the `ChannelModal` immediately, but pass it it's loading parameter, so that we know its doing something.